json-lint/schemas/inventory/adservers/appnexus.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$id": "#appnexus",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Appnexus adserver",
+    "description": "https://github.com/ampproject/amphtml/blob/master/ads/appnexus.md",
+    "additionalProperties": false,
+    "required": [
+        "type"
```

Comment:
Only type required? Hmmmm

Reply:
Added three new fields and one of them is mandatory at least:
https://github.com/ampproject/amphtml/blob/master/ads/appnexus.md